config/locales/service_providers/fr.yml

```diff
@@ -28,3 +28,7 @@ fr:
       body_html: Si vous avez déjà un profil SAM, veuillez utiliser l'adresse e-mail
         principale ou secondaire que vous avez utilisée pour SAM pour %{link}.
       create_account_link: créer votre nouveau compte login.gov
+      create_account_page:
+        body: Si vous avez déjà un profil SAM, veuillez utiliser l'adresse e-mail
+          principale ou secondaire que vous avez utilisée pour SAM pour créer votre
+          nouveau compte login.gov.
```

Comment:
Why do the French and Spanish translations refer to "primary or secondary" email address, whereas the English test does not?

Reply:
I was supposed to use the text from account_page.body for create_account_page.body but it appears the account_page.body for spanish and french are currently wrong in production.  I need new translations for both.